src/components/ItemDetail/SingleMetadata.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SingleMetadata = props => {
+  const { title, items } = props;
+
+  if (items) {
+    return (
+      <div>
+        <h4>{title}</h4>
+        <p>{items}</p>
+      </div>
+    );
+  } else {
+    return null;
```

Comment:
Sometimes React will complain about a `null` value for a situation like this.  A more common React convention is just to return an empty string if the output is markup.